### PR TITLE
Update revalidatePath.mdx

### PR DESCRIPTION
**Pull request by krychaxp**
add if statement, because typescript throws error: revalidatePath require `string`, but `searchParams.get('path')` returns `string|null` type

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e286b806841961c084d2e25955f6aaec6abf53c2

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/04-functions/revalidatePath.mdx [hunk 1]
```diff
@@ -41,13 +41,22 @@ export default async function submit() {
...
+
+  if (path) {
+    revalidatePath(path);
+    return NextResponse.json({ revalidated: true, now: Date.now() });
+  }
+
+  return NextResponse.json({
+    revalidated: false,
+    now: Date.now(),
+    message: "Missing 'path'",
```
Comment (leerob) [inline docs/02-app/02-api-reference/04-functions/revalidatePath.mdx:58]:
```suggestion
    message: 'Missing path to revalidate',
```

**Review by leerob — DISMISSED** [review]
Could you add these changes the JS codeblock below as well?

**Comment by krychaxp** [comment]
> Could you add these changes the JS codeblock below as well?

added :D

**Review by ijjk — APPROVED** [review]

**Pull request merged**